### PR TITLE
Article/appmod 101

### PR DESCRIPTION
**Pull request by aj-mechrock**
Application Modernisation 101 discussion. the six R's

### CODE REVIEW

File: _posts/2021-03-10-application-modernisation-101.markdown [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+## Retain
+Well, this isn’t application modernisation at all. But always ask yourself, “Is the way we’re doing it currently the best way for our current needs”. If the answer is, “Yes”, then why are you changing anything?
+
+## Retire
+Do you really need the application? Particularly in large organisations, or organisations that have experienced rapid growth, duplicate capabilities are often introduced into the mix. Do you have duplicated application capabilities and would you be best served consolidating those applications and retiring some applications? No only are you reducing the amount of administration overhead, potentially there’s a reduction in licensing, training costs and data integration efforts.
```
Comment (Hamo69) [inline _posts/2021-03-10-application-modernisation-101.markdown:26]: > No only are you

should be 

> Not only are you

File: _posts/2021-03-10-application-modernisation-101.markdown [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+## Replatform
+Replatforming your application requires strategic and selective migration of services to managed services offerings. This means undertaking tasks such as migration from databases running on on-prem servers to managed services like RDS and applications to container-based infrastructure. This is a solid approach that minimises application redevelopment and still started to benefit from the scalability, redundancy and fault tolerance capabilities of cloud based infrastructure. It does require more effort ($$$) than simple rehosting, however the total cost of ownership savings are significant.
```
Comment (Hamo69) [inline _posts/2021-03-10-application-modernisation-101.markdown:35]: > and still started to benefit 

could be

> but starts to benefit

File: _posts/2021-03-10-application-modernisation-101.markdown [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+## Refactor
```
Comment (Hamo69) [inline _posts/2021-03-10-application-modernisation-101.markdown:37]: Can you link to Tim's Strangler Pattern blog?

File: _posts/2021-03-10-application-modernisation-101.markdown [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
...e refactoring your application. This can range from adopting the Strangler Pattern and methodically replacing and rewriting parts of your application, through to complete wholesale rewrites. The aim with refactoring your application, will be to leverage Platform as a Service components of your chosen cloud provider to get the best resilience, scalability and performance. Refactoring your application to use serverless technologies will see great improvements in its efficiency, where you’re only paying for what you use, and are able to scale up effortlessly. Critics of this Cloud Native will often quote vendor lock-in as the major issue with this approach, and this is true - you will be tied to the platform provider’s implementation of particular services, however… this is also true of any software development project using any framework, whether cloud native or not. As with replatforming, refactoring requires significant effort and will often have the largest outlays of any of the application modernisation strategies, however it also provides the biggest cost savings in the long term, and positions the application in the best position for future maintenance, enhancement and scaling.
```
Comment (Hamo69) [inline _posts/2021-03-10-application-modernisation-101.markdown:38]: Could we use the SmartApps stats that Tim just got for Dept of Communities?

File: _posts/2021-03-10-application-modernisation-101.markdown [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+|Fault Tolerance|Medium|High|High|
+|Availability|Medium|High|High|
+|Skills availability|High|Low|Medium|
+
+** Please note, these categorisations are indicative and relative that depend completely on the Application/Workload. There are a multitude of exceptions to these rules of thumb.
+
+## Conclusion
+The landscape of digital infrastructure is changing - your challenge is to decide how to leverage it for the benefit of your organisation There’s no one right way to move your applications into 2021. The only failure would be to not make a decision at all. 
+
+The approach you take will vary depending on your individual circumstances, your budget and your timeline. Fortunately, there is a spectrum of solutions available, each with their own pros and cons. The real benefits of cloud are only fully realised as you trend towards the refactor/rebuild end of the spectrum. Hopefully this article provides some insight as to what your options are. Stay tuned to our blog for an in-depth look into replatforming, coming soon.
```
Comment (Hamo69) [inline _posts/2021-03-10-application-modernisation-101.markdown:71]: want to throw in this link at the end as a CTA?

https://www.mechanicalrock.io/our-expertise/application-modernisation

**Review by Hamo69 — APPROVED** [review]

File: _posts/2021-03-10-application-modernisation-101.markdown (hunk 4 above)
Comment (aj-mechrock) [reply _posts/2021-03-10-application-modernisation-101.markdown:38]: yep

**Comment by aj-mechrock** [comment]
Aiming to merge this on Blogsday.. (tuesday)

**Pull request merged**